ProcessLib/RichardsMechanics/RichardsMechanicsFEM-impl.h

```diff
@@ -824,6 +878,67 @@ void RichardsMechanicsLocalAssembler<ShapeFunctionDisplacement,
         .noalias() += Kup * p_L;
 }
 
+template <typename ShapeFunctionDisplacement, typename ShapeFunctionPressure,
+          typename IntegrationMethod, int DisplacementDim>
+std::size_t RichardsMechanicsLocalAssembler<
+    ShapeFunctionDisplacement, ShapeFunctionPressure, IntegrationMethod,
+    DisplacementDim>::setKelvinVector(double const* values,
+                                      KelvinVectorType IpData::*member)
+{
+    auto const kelvin_vector_size =
```

Comment:
In `getSigma()` the `kelvin_vector_size` is a 
```constexpr int kelvin_vector_size = MathLib::KelvinVector::KelvinVectorDimensions<DisplacementDim>::value;```
Why here a different implementitation?